assets/src/application/ui/display/Tooltip/types.ts

```diff
@@ -1,4 +1,6 @@
+import HelpIcon from '@appDisplay/icons/HelpIcon';
+
 export interface ClickableIconWithTooltipProps {
-	icon: React.ComponentType;
+	icon: typeof HelpIcon;
```

Comment:
interesting, so here other icons also should be added

Reply:
You can also use `React.ComponentType<IconProps>`

Reply:
Thanks for the suggestion :bow: